src/worker.js

```diff
@@ -51,11 +51,6 @@ if (ENVIRONMENT_IS_NODE) {
 }
 #endif // ENVIRONMENT_MAY_BE_NODE
 
-// Thread-local:
-#if EMBIND
-var initializedJS = false; // Guard variable for one-time init of the JS state (currently only embind types registration)
```

Comment:
I think this maybe has to stay here in this file, since code in the module object is not available until later after the module is loaded.   At the very least if you move if you will need to access via the Module object (at least in the general case).

Reply:
I'm still confused about how code is shared between worker.js and the rest because `executeNotifiedProxyingQueue` works fine (AFAICT) without being accessed through `Module`.

Reply:
Did you try running with `-s MODULARIZE` .. I think we had a bug report about that not working for `executeNotifiedProxyingQueue`

Reply:
Oh yeah, that's totally broken. Why isn't that caught by tests?

Reply:
I'm guess just lack a bunch of coverage for MODULARIZE+pthreads.

Reply:
And would you prefer that I fix it in a separate PR or this one?

Reply:
I pushed a commit to fix it now, but there are still no tests. Maybe I can add a MODULARIZE test in a follow-up?

Reply:
Up to you... as long we get more coverage in the long run.   Might be cleaner to keep this change focused.

Reply:
Sounds good. Reverted those changes for now.